lib/util/style.dart

```diff
@@ -507,18 +516,95 @@ Divider addDialogDivider({double height = 32}) {
 }
 
 Widget get autonomyLogo {
+  return FutureBuilder<Pair<bool, bool>>(
+      future: logoState(),
+      builder: (context, snapshot) {
+        if (snapshot.data == null) return const SizedBox(height: 50);
+        return Row(
+          crossAxisAlignment: CrossAxisAlignment.start,
+          children: [
+            SvgPicture.asset(
+              snapshot.data!.first == true
+                  ? "assets/images/logo_dev.svg"
+                  : "assets/images/penrose_moma.svg",
+              width: 50,
+            ),
+            const SizedBox(width: 15),
+            snapshot.data!.second
+                ? proLabel(Theme.of(context))
+                : const SizedBox(),
+          ],
+        );
+      });
+}
+
+Widget get autonomyWhiteLogo {
   return FutureBuilder<bool>(
-      future: isAppCenterBuild(),
+      future: isPremium(),
       builder: (context, snapshot) {
-        return SvgPicture.asset(
-          snapshot.data == true
-              ? "assets/images/logo_dev.svg"
-              : "assets/images/penrose_moma.svg",
-          width: 50,
+        if (snapshot.data == null) return const SizedBox(height: 50);
+        return Row(
+          crossAxisAlignment: CrossAxisAlignment.start,
+          children: [
+            SvgPicture.asset(
+              "assets/images/autonomy_icon_white.svg",
+              width: 50,
+              height: 50,
+            ),
+            const SizedBox(width: 15),
+            snapshot.data!
+                ? proLabel(Theme.of(context), isWhite: true)
+                : const SizedBox(),
+          ],
         );
       });
 }
 
+Widget header(double paddingTop, {isWhite = false}) {
```

Comment:
this file is actually only designed to store custom styles. For new Widgets, I suggest move them into a new class (StatelessWidget) under `view` folder.